docker/docker-compose.yml

```diff
@@ -19,11 +19,14 @@ services:
       #user should not edit container paths
```

Comment:
When you submit PRs, you should not modify the docker-compose.yml.  The default HEAT yml does not have to be added to the commit when you are preparing